src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/_article_page.scss

```diff
@@ -158,6 +158,37 @@ main article {
         }
       }
 
+      .subjects {
+        > div {
+          width: 100%;
+
+          // flex box layout can be left-right with wrapping, cannot do top-down columns
+          -webkit-column-count: 2;
```

Comment:
I think there is a kube way to do this. Avoids all the webkit, moz prefixes

Reply:
if you can find example please let me know.